src/freertos_drivers/st/Stm32Can.cxx

```diff
@@ -520,6 +589,24 @@ void can1_sce_interrupt_handler(void)
     Stm32Can::instances[0]->sce_interrupt_handler();
 }
 
+#ifdef CAN2
+void usb_hp_can2_tx_interrupt_handler(void)
```

Comment:
these are not correctly named; they should be the same as the ones below.
It might make sense to have the CAN2 and CAN3 interrupts outside of the major ifdef around the stm32 chip type, since they are always named the same.

Reply:
Fixed.